package.json

```diff
@@ -35,7 +35,15 @@
     "yargs": "^15.4.1"
   },
   "overrides": {
-    "glob": "^9.3.5",
+    "copyfiles": {
```

Comment:
when overriding transitive deps, we still use "^9.3.5" instead of "^11.0.3", for maximum compatibility